scripts/test-truncating-of-client-hello.py

```diff
@@ -188,7 +188,7 @@ def run_all(conversations):
         try:
             runner.run()
         except:
-            # TBD: print(traceback.format_exc())
+            print(traceback.format_exc())
```

Comment:
the problem is that this traceback will be detached from the name of the test case, as names are printed at the end of execution now